### PR TITLE
Dockerfile.rocm.ubi: bump ROCm to 6.2

### PR DESCRIPTION
**Pull request by dtrifiro**

### CODE REVIEW

**Comment by dtrifiro** [comment]
Currently failing with:

```
 > [rocm_devel 1/2] RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm &&     rpm -ql epel-release &&     microdnf -y update &&     microdnf -y install         ccache         git         rocm         hipcc         wget         which &&     microdnf clean all:
16.63 Nothing to do.
16.65
16.65 (microdnf:127): librhsm-WARNING **: 16:31:31.236: Found 0 entitlement certificates
16.65
16.65 (microdnf:127): librhsm-WARNING **: 16:31:31.237: Found 0 entitlement certificates
16.74 error: Could not depsolve transaction; 1 problem detected:
16.74  Problem: package rocm-6.2.0.60200-66.el9.x86_64 from ROCm-6.2 requires mivisionx = 3.0.0.60200-66, but none of the providers can be installed
16.74   - package mivisionx-3.0.0.60200-66.x86_64 from ROCm-6.2 requires rocdecode, but none of the providers can be installed
16.74   - conflicting requests
16.74   - nothing provides libva needed by rocdecode-0.6.0.60200-66.x86_64 from ROCm-6.2
```

@RH-steve-grubb do you have any pointers?

**Comment by RH-steve-grubb** [comment]
pull it from centos? https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/libva-2.20.0-1.el9.x86_64.rpm

**Comment by dtrifiro** [comment]
Waiting on https://github.com/ROCm/ROCm/issues/3957

**Comment by dtrifiro** [comment]
Update: explicitly added the set of packages required in the build step to avoid the above `rocm` package issue, built is now succesfull and the image does seem to be working fine.

File: Dockerfile.rocm.ubi [hunk 1]
```diff
@@ -144,11 +150,12 @@ ENV VLLM_INSTALL_PUNICA_KERNELS=1
 RUN --mount=type=cache,target=/root/.cache/ccache \
     --mount=type=cache,target=/root/.cache/pip \
     --mount=type=cache,target=/root/.cache/uv \
+    ln -s /opt/vllm/lib64/python3.12/site-packages/torch/lib/libroctx64.so /opt/rocm/lib/libroctx64.so && \
```
Comment (dtrifiro) [inline Dockerfile.rocm.ubi:153]: This fixes a cmake issue due to conflicting deps (cmake lookin in /opt/rocm/lib but libroctx64.so being shadowed by `/opt/vllm/...`

**Pull request merged**